packages/nimbus/src/components/tooltip/tooltip-dev.mdx

```diff
@@ -0,0 +1,188 @@
+
+```jsx-live-dev
+// Example: A basic tooltip
+const App = () => (
+<Tooltip.Root placement="top">
+  <Button>Hover me</Button>
+    <Tooltip.Content
+      placement="top"
+      maxWidth="300px"
+      backgroundColor="blue"
+      color="white"
+    >
+      This is a helpful tooltip
+    </Tooltip.Content>
+</Tooltip.Root>
+)
+```
+
+### Common props and Variations
+
+#### Placement Options
+Control where the tooltip appears relative to its trigger: change the placement to any of these options to reposition the tooltip:
+```"top" | "top left" | "top right" | "top start" | "top end" | "bottom" | "bottom left" | "bottom right" | "bottom start" | "bottom end" | "left" | "left top" | "left bottom" | "right" | "right top" | "right bottom" | "start" | "start top" | "start bottom" | "end" | "end top" | "end bottom"```
+
+```jsx-live-dev
+const App = () => (
+<Tooltip.Root>
+      <Button>Tooltip placement</Button>
+      <Tooltip.Content placement="top">Tooltip on top</Tooltip.Content>
+</Tooltip.Root>
+)
+```
+
+#### Offset Options
+
+Control the tooltip offset:
+```offset```
+
+```jsx-live-dev
+const App = () => (
+<Tooltip.Root>
+      <Button>Tooltip placement</Button>
+      <Tooltip.Content offset={50}>Tooltip on top</Tooltip.Content>
+</Tooltip.Root>
+)
+```
+
+```....Add example snippets for each relevant component prop```
+
+### API Changes
+
+| UI Kit Component | Nimbus Component | Notes |
+|------------------|------------------|-------|
+| `<Tooltip>` | `<Tooltip.Root>` + `<Tooltip.Content>` | Now uses compound component pattern |
+| `title` prop | `<Tooltip.Content>` children | Text content now goes as children |
+| `isOpen` prop | `isOpen` prop | ✅ Same API |
+| `onClose` | `onOpenChange` | ⚠️ Renamed, now receives boolean parameter |
+| `horizontalConstraint` | Not supported | ❌ Removed - use CSS or container sizing |
+
+
+#### Deprecated props
+- `onClose` → `onOpenChange`: Now receives `(isOpen: boolean) => void` instead of `() => void`
+- `title` prop → Content as children: Text now goes inside `<Tooltip.Content>`
+- `delay` and `closeDelay`: More granular control over timing
+- `horizontalConstraint`: Use CSS width constraints on content instead
+- `tone`: Styling is now handled through design tokens
+
+## Testing Strategies and Best Practices
+
+#### 1. Unit Testing with React Testing Library
+
+**Recommended Testing Library Setup:**
+```tsx
+import { render, screen, userEvent } from '@testing-library/react';
+import { NimbusProvider } from '@commercetools/nimbus';
+import { IntlProvider } from 'react-intl';
+import { Tooltip, Button } from '@commercetools/nimbus';
+
+// Test wrapper with required providers
+const TestWrapper = ({ children }: { children: React.ReactNode }) => (
+  <IntlProvider locale="en" messages={{}} defaultLocale="en">
+    <NimbusProvider>
+      {children}
+    </NimbusProvider>
+  </IntlProvider>
+);
```

Comment:
Aren't we going to put `NimbusProvider` into App Kit? One of Tobi's biggest complaints was this here pattern in tests.

Reply:
> Do you have any examples? Do the design guidelines suffice?

Yes, I think we can point to the guidelines here